### DISCUSSION TITLE
shared Scene System with PUN

### DISCUSSION THREAD

**Discussion by ttkrpink** [microsoft/MixedRealityToolkit-Unity#11292] [Ideas]
Examples of the scene system work fine with a single user.  When used in a shared environment, PUN has its own scene-loading function. In that case, it seems no example of an MRTK shared scene system is available, that combines the MRTK scene system and the PUN scene system.

Is it possible for the MS MRTK team to show me an example or a tutorial?

Thanks.

**Comment by Zee2** [comment]
We don't have any examples for Photon, off the top of my head, no... You can take a peek at my hobby project over here (https://github.com/Zee2/MRTK3-Netcode) that does MRTK3 multi-user with the new Netcode for GameObjects. Not exactly 1:1 relevant I know, but possibly worth a look?

Moving this into a Q&A for now so other folks can respond with any suggestions.
